R/git-default-branch.R

```diff
@@ -566,11 +582,14 @@ fishy_github_actions <- function(new_name = "main") {
   }
 
   paths <- sort(paths)
-  ui_paths <- map_chr(paths, ui_path)
-
-  ui_oops(c(
-    "These GitHub Action files don't mention the new default branch {ui_value(new_name)}:",
-    paste0("- ", ui_paths)
+  ui_paths <- map_chr(paths, ui_path_impl)
+
+  # TODO: the ui_paths used to be a nested bullet list
+  # if that ever becomes possible/easier with cli, go back to that
```

Comment:
It would be nice to have nested bullets in `cli::cli_bullets()`. But AFAICT that's only possible if you're making a bullet list more "by hand" with `cli_ol()`, `cli_ul()`, `cli_li()`, and the like.